src/main/java/com/sweng894/GetVaccinated/api/repository/AppointmentRepository.java

```diff
@@ -0,0 +1,40 @@
+package com.sweng894.GetVaccinated.api.repository;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapper;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBSaveExpression;
+import com.amazonaws.services.dynamodbv2.model.AttributeValue;
+import com.amazonaws.services.dynamodbv2.model.ExpectedAttributeValue;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+import com.sweng894.GetVaccinated.api.entity.Appointment;
+
+@Repository
+public class AppointmentRepository {
```

Comment:
This class lacks test coverage, but I can understand why. It needs access to DynamoDB to run.

We could eventually solve this with having an integration test that uses local dynamodb, but I think it's okay to take a hit to the code coverage percentage metrics for now.